src/coreclr/tests/src/JIT/HardwareIntrinsics/Arm/AdvSimd.Arm64/DuplicateToVector64.Int64.cs

```diff
@@ -0,0 +1,300 @@
+        {
+            private byte[] outArray;
+
+            private GCHandle outHandle;
+
+            private ulong alignment;
+
+            public DataTable(Int64[] outArray, int alignment)
+            {
+                int sizeOfoutArray = outArray.Length * Unsafe.SizeOf<Int64>();
+                if ((alignment != 16 && alignment != 8) || (alignment * 2) < sizeOfoutArray)
+                {
+                    throw new ArgumentException("Invalid value of alignment");
+                }
+
+                this.outArray = new byte[alignment * 2];
+
+                this.outHandle = GCHandle.Alloc(this.outArray, GCHandleType.Pinned);
+
+                this.alignment = (ulong)alignment;
+            }
+
+            public void* outArrayPtr => Align((byte*)(outHandle.AddrOfPinnedObject().ToPointer()), alignment);
+
+            public void Dispose()
+            {
+                outHandle.Free();
+            }
+
+            private static unsafe void* Align(byte* buffer, ulong expectedAlignment)
+            {
+                return (void*)(((ulong)buffer + expectedAlignment - 1) & ~(expectedAlignment - 1));
+            }
+        }
+
+        private struct TestStruct
+        {
+            public Int64 _fld;
+
+            public static TestStruct Create()
+            {
+                var testStruct = new TestStruct();
+                testStruct._fld = TestLibrary.Generator.GetInt64();
+                return testStruct;
+            }
+
+            public void RunStructFldScenario(DuplicateUnaryOpTest__DuplicateToVector64_Int64 testClass)
+            {
+                var result = AdvSimd.Arm64.DuplicateToVector64(_fld);
+                Unsafe.Write(testClass._dataTable.outArrayPtr, result);
+                testClass.ValidateResult(_fld, testClass._dataTable.outArrayPtr);
+            }
+        }
+
+        private static readonly int LargestVectorSize = 8;
+
+        private static readonly int RetElementCount = Unsafe.SizeOf<Vector64<Int64>>() / sizeof(Int64);
+
+        private static Int64 _data;
+
+        private static Int64 _clsVar;
+
+        private Int64 _fld;
+
+        private DataTable _dataTable;
+
+        static DuplicateUnaryOpTest__DuplicateToVector64_Int64()
+        {
+            _clsVar = TestLibrary.Generator.GetInt64();
+        }
+
+        public DuplicateUnaryOpTest__DuplicateToVector64_Int64()
+        {
+            Succeeded = true;
+
+            _fld = TestLibrary.Generator.GetInt64();
+            _data = TestLibrary.Generator.GetInt64();
+
+            _dataTable = new DataTable(new Int64[RetElementCount], LargestVectorSize);
+        }
+
+        public bool IsSupported => AdvSimd.Arm64.IsSupported;
+
+        public bool Succeeded { get; set; }
+
+        public void RunBasicScenario_UnsafeRead()
+        {
+            TestLibrary.TestFramework.BeginScenario(nameof(RunBasicScenario_UnsafeRead));
+
+            var result = AdvSimd.Arm64.DuplicateToVector64(
```

Comment:
>AdvSimd.Arm64.DuplicateToVector64 [](start = 25, length = 33)

We don't have `AdvSimd.Arm64.DuplicateToVector64()` so you don't need this. I am not sure how CI is not complaining. Did you make sure that CI picked up these newly added test?

Reply:
This was:
* Original change had `DuplicateToVector64`
* Realized it was a typo from API review and isn't something we are exposing
* Tests were regenerated, but the `DuplicateToVector64` tests from the last generation weren't deleted

Reply:
Got it. Thanks!

Reply:
Removed